docs/builders/models/model-loading.md

```diff
@@ -7,139 +7,99 @@ parent: Models
-    s 1
-    usemtl white
-    f 1216/1216/491 1217/1217/492 1218/1218/493
-    f 1216/1216/491 1219/1219/494 1217/1217/492
-
-    ...
+~~~lua
+    GeometryFile = asset.localResource("nyc-model.obj"),
 ~~~
 
-7. Switch to the material file. Here is where you define your materials and the textures. Create a new material with: <code>newmtl materialName</code>. Note that materialName should be the same as you specified in the model file in step 4. Then connect the material to a texture using: <code>map_Kd textureName.png</code>. Note that the path to the texture should be given relative to the material file.
-
-8. If you specified several **different** materials in step 5 you will need to repeat step 7 for every new material. 
+The <code>localResource</code> function refers to a file that lies right next to the asset file on the filesystem. If you want to reference a file on your computer that is not located directly next to the asset file, you can instead give the full path to that file like this:
 
-9. In the end your material file should look something like this (with two materials):
+~~~lua
+    GeometryFile = "C:/Users/username/Documents/data/nyc-model.obj",
 ~~~
-    newmtl ISS_03_dull
-        map_Kd 0.png
 
-    newmtl white
-        map_Kd white_20.png
-~~~
+Note that the slashes in the path need to be forward slashed (<code>/</code>) and not backward slashes (<code>\\</code>); double backward slashes would also work (<code>\\\\</code>). There are additional properties you can set for your model, such as scale and animation, to read more about these see [Model Scale](../models/model-scale), and [Animated Models](../models/model-animation).
 
-10. Make sure that your material file is in the same directory as the model file. Your model should now be textured in OpenSpace.
+## Formats
+OpenSpace uses the [Assimp library](https://github.com/assimp/assimp) to load models; therefore, our supported formats are similar to their supported formats. For a complete list see [List of formats](#list-of-formats) further down this page.
 
 ## Debugging your model
-If your model does not show up in OpenSpace and you are sure that you have done everything right there is a new tool that you could use for debugging. In the asset file you can add an optional property for forcing invisible parts of the model to render. This forces any part of the model that is invisible (has no texture or color) to render. This property is called <code>ForceRenderInvisible</code>, for example the Juno spacecraft:
+If your model does not show up in OpenSpace and you are sure that you have done everything right in the loading, there is a tool that you could use for debugging. In the asset file, you can add an optional property for forcing invisible pieces of the model to render. This forces any part of the model that is invisible (has no texture or color) to render. This property is called <code>ForceRenderInvisible</code>, for example the Juno spacecraft:
 
-~~~
+~~~lua
```

Comment:
Nitpicky, but I would change the indentation of all the Lua snippets to use 2 spaces instead of 4, to match our asset code style